### PR TITLE
SITE-6625 adding Free svg icon

### PR DESCRIPTION
**Pull request by msied**
screen shot 2015-11-11 at 12 04 34 pm

### CODE REVIEW

**Comment by msied** [comment]
@kevzettler does this look right?

**Comment by tildedave** [comment]
Use defineSvg here - @cohitre can give more pointers

**Comment by msied** [comment]
How's this, @tildedave ?

File: app/components/Free.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+const React = require('react');
+const { defineSvg } = require('../defineSvg');
+
+module.exports = defineSvg({
+  className: 'tilt-icon-tag-free',
+
+  render() {
+    return (<svg className={this.getClassNames()} viewBox="0 0 64 64">
+      <title>Free</title>
+      <g stroke="none" strokeWidth="1" fill="none" fillRule="evenodd">
```
Comment (tildedave) [inline app/components/Free.js]: these styles are best in CSS - see similar other recent PRs for best practices around this.  https://github.com/Crowdtilt/tilt-images/pull/35/files is a decent prototype

File: app/components/Free.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+        <g fill="#00A0D8">
```
Comment (tildedave) [inline app/components/Free.js]: should be in CSS

**Comment by kevzettler** [comment]
update 
https://github.com/Crowdtilt/tilt-images/blob/master/Changelog.md

With the latest version number and notes on your update.

**Comment by kevzettler** [comment]
When this gets merged you'll head to 

https://github.com/Crowdtilt/tilt-images/releases

and click "Draft A New Release"

When the new release is published update

https://github.com/Crowdtilt/crowdtilt-public-site/blob/dev/package.json#L70

With the release version

**Comment by kevzettler** [comment]
lgtm when :green_apple:

**Comment by cohitre** [comment]
Also needs to update the version in package.json

File: Changelog.md [hunk 3]
```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## 4.1.1
```
Comment (tildedave) [inline Changelog.md]: this should be 4.2.0

**Comment by tildedave** [comment]
ping, 4.2.0 not 4.1.1

**Comment by kevzettler** [comment]
https://github.com/Crowdtilt/tilt-images/blob/master/package.json#L4

**Comment by cohitre** [comment]
If you add this file to app/components.js we'll be able to run `renders without errors` tests on it.

**Comment by tildedave** [comment]
yeah, this needs to be added to the demo page too

**Comment by msied** [comment]
ping

**Comment by msied** [comment]
ping

**Comment by kevzettler** [comment]
lgtm

**Pull request merged**